android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -570,7 +572,7 @@ private void hidePositionChooser()
   {
     UiUtils.hide(mPointChooser);
     Framework.nativeTurnOffChoosePositionMode();
-    setFullscreen(false);
```

Comment:
The previous name was clearer about the behavior. Why did you decide to change it?

Reply:
The previous `setFullscreen` implementation simply hid the map buttons, which is not really a fullscreen mode. The new `setFullscreen` also hides system bars to create a real fullscreen. But in the case of the position chooser, we only want to hide the map buttons but keep the system bars.